US-CBP/cbp-theme
# Feature/add css with no fonts

#### What's this PR do?

Adds an additional set of min and non min css that do not bundle the roboto and font awesome css

- cbp-theme-no-fonts.css
- cbp-theme-no-fonts.min.css

#### Where should the reviewer start?

Take a look at the package.json and follow to the build/sass-builder-no-fonts.js

#### How should this be manually tested?

1. Run npm run build
2. Verify that the new css files do not have font-awesome and Roboto css bundled
3. Run npm run dev verify original functionality works

#### Any background context you want to provide?

- This allows for folks to supply their own css for font-awesome and roboto, perhaps using a CDN

#### What are the relevant tickets?

#264 #267

#### Screenshots (if appropriate)
#### Questions:
- Is there a blog post? N/A
- Does the knowledge base need an update? In general, yes, it needs an overhaul but not part of this PR
